### DISCUSSION TITLE
[SUGGESTION] Wiki -> MAME samples folder

### DISCUSSION THREAD

**Discussion by tarikalmashni** [OnionUI/Onion#511] [Q&A] [answered]
You should add to the **ARCADE** paragraph in the emulator wiki where to place the samples folder that comes with MAME rom sets.
I don't have my Miyoo Mini yet but I guess they go here:
[RetroArchBiosFolder]/mame2003-plus/samples/[zips in here]
At least I would put them there according to the official Libretro docs (don't have my own Mini yet, so can't verify): https://docs.libretro.com/library/mame2003_plus/#audio-samples

**Comment by dwmccoy** [comment]
You are indeed correct.  The Audio Samples for Mame 2003Plus should be placed into the /BIOS/mame2003-plus/samples folder from the ROOT of the SDCARD.

**Comment by Olywa** [comment, marked as answer]
This has now been added to the Wiki, thank you both.
